it/mail/chrome/messenger/accountCreation.dtd

```diff
@@ -60,27 +60,27 @@
 <!ENTITY changeSettings.label            "Cambia impostazioni">
 <!ENTITY changeSettings.accesskey        "m">
 
-<!ENTITY pb.contactYourProvider.description "&brandShortName; can allow you to get to your mail using the provided configurations. However, you should contact your administrator or email provider regarding these improper connections.">
+<!ENTITY pb.contactYourProvider.description "&brandShortName; può consentirti di collegarti alla tua e-mail utilizzando le configurazioni fornite. Tuttavia, dovresti contattare il tuo amministratore o il tuo fornitore di servizi e-mail in merito a queste connessioni non corrette.">
 
 <!ENTITY insecureServer.tooltip.title    "Attenzione, questo è un server non sicuro.">
-<!ENTITY insecureServer.tooltip.details  "Clic sul cerchio per maggiori informazioni.">
+<!ENTITY insecureServer.tooltip.details  "Clicca sul cerchio per maggiori informazioni.">
 
-<!ENTITY insecureUnencrypted.description "La posta e le informazioni di autenticazione vengono trasmesse senza cifratura, pertanto le password e i messaggi possono essere facilmente letti da altre persone. &brandShortName; consentirà la lettura della posta ma si consiglia di contattare il provider per configurare il server con un collegamento sicuro.">
-<!ENTITY insecureSelfSigned.description  "Non è possibile verificare il certificato utilizzato dal server, pertanto non si può escludere che qualcuno stia intercettando il traffico tra &brandShortName; e il server. &brandShortName; consentirà la lettura della posta ma si consiglia di convincere il provider a fornire un collegamento sicuro con il server.">
+<!ENTITY insecureUnencrypted.description "La tua e-mail e le tue informazioni di autenticazione sono inviate senza crittografia, pertanto la tua password e il tuo messaggio potrebbero essere facilmente letti da altri utenti. &brandShortName; ti consentirà di collegarti alla tua e-mail ma dovresti contattare il tuo fornitore di servizi e-mail per configurare il server con un certificato affidabile.">
```

Comment:
CORRECTIONS NEEDED. Too many possessive pronouns are redundant. I would suggest to modify the sentence as follows: 

 +<!ENTITY insecureUnencrypted.description "La tua e-mail e le informazioni di autenticazione sono inviate senza crittografia, pertanto sia la password che il messaggio potrebbero essere facilmente letti da altri utenti. &brandShortName; consentirà di collegarti alla tua e-mail, ma dovresti contattare il fornitore di servizi e-mail per configurare il server con un certificato affidabile.">